platform-sdk/docs/core/address-book-management.md

```diff
@@ -95,7 +95,7 @@ The `AddressBook` supports the following queries:
 - `int getIndex(long id)`: get the index in this address book for a given node ID
 - `Address getAddress(long id)`: given a node ID, return that node's address
 
-TODO
```

Comment:
The string "TODO" in any form is annoying. I've got an intellij window that shows me all of the TODOs in our code, and I like keeping it at zero for merge ready branches.